internal/server/server.go

```diff
@@ -21,16 +22,29 @@ import (
 // StartServer start Authelia server with the given configuration and providers.
 func StartServer(configuration schema.Configuration, providers middlewares.Providers) {
 	autheliaMiddleware := middlewares.AutheliaMiddleware(configuration, providers)
-	publicDir := os.Getenv("PUBLIC_DIR")
-	if publicDir == "" {
-		publicDir = "./public_html"
+	publicDir := "/public_html"
+
+	if os.Getenv("LOCAL_ASSETS") == "true" {
```

Comment:
What do you think about removing PUBLIC_DIR and only keeping LOCAL_ASSETS env var containing the path? If it's not provided or empty we take the embedded files. It's just to avoid stacking env vars.
For old setups having the PUBLIC_DIR env var after the bump, we can easily and safely display a warning saying that setting this env var is not doing anything anymore.

We can go even further by moving this config from env var into the config file. It's my fault if env vars were introduced at the beginning and there was no good reason to do that, my bad.

Reply:
Agree with this. Perhaps name it to "ASSETS_DIR" and just document the behaviour in the README.

Also not sure if everyone agrees with this, but I generally only use environment variables for sensitive info (passwords, etc.) because they usually don't show in process explorers. If everything else can go in a config file/CLI/some other way, I try to do it.

Sorry to have a complete stranger butt in - /2c

Reply:
> For old setups having the PUBLIC_DIR env var after the bump, we can easily and safely display a warning saying that setting this env var is not doing anything anymore.
> 
> We can go even further by moving this config from env var into the config file. It's my fault if env vars were introduced at the beginning and there was no good reason to do that, my bad.

I've taken the approach of adding an `Assets` key in the root configuration alongside `Host` and `Port`.

If not provided/empty it will utilise the embedded assets, otherwise it will utilise the path provided to attempt to load assets.